astro/src/components/docs/release-notes/ThemeUpdateWarning.astro

```diff
@@ -0,0 +1,11 @@
+---
+import Aside from 'src/components/Aside.astro';
+---
+
+<Aside type="theme" title="Theme update">
+<p>This version of FusionAuth includes changes to the default FusionAuth theme. If you have created custom themes in previous versions of FusionAuth, <strong>you will need to incorporate these changes into your custom themes</strong>.</p>
```

Comment:
Not sure if this is always true? 

> <strong>you will need to incorporate these changes into your custom themes</strong>.</p>

Sometimes it will be, and other times it may be, "we added a page and it won't affect you at all unless you turn on some feature". 

Ideally this boiler plate portion would say that we made a change that may affect you if you have one or more custom themes. Please read below to understand if this affects you, or if you need to take immediate action prior to upgrading.

Reply:
I was trying to scare them into _actually_ reading the notes. I'll take another try at phrasing.

Reply:
Trying out
> This version of FusionAuth includes changes that may affect you if you have one more custom themes. **Please read below to understand if this affects you**, or if you need to take immediate action prior to upgrading.